assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/index.js

```diff
@@ -89,32 +96,35 @@ const DashboardIdeasWidget = ( { Widget } ) => {
 							focusOnActivate={ false }
 						>
 							{ __( 'New', 'google-site-kit' ) }
+							{ newIdeas?.length >= 0 && <span>({ newIdeas.length })</span> }
```

Comment:
This is not needed for the new ideas tab:

> The "Saved" and "Drafts" tab labels should be enhanced so that they include the total number of saved ideas / draft ideas available.

Reply:
Good catch! Updated!